### PR TITLE
Templatising the StartKit

### PR DESCRIPTION
**Pull request by CartBlanche**

### CODE REVIEW

File: Meadow_DotNet_SDK/Project_Templates/templates/Meadow.StartKit/StartKit.Core/.template.config/template.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "classifications": [ "Meadow", "Library" ],
+  "name": "Meadow StartKit (Core)",
+  "identity": "WildernessLabs.Meadow.StartKit.Core",
+  "groupIdentity": "WildernessLabs.Meadow.SKC",
+  "shortName": "MeadowSKC",
+  "tags": {
+    "language": "C#",
+    "type": "project"
+  },
+  "sourceName": "$safeprojectname$",
```
Comment (CartBlanche) [inline Meadow_DotNet_SDK/Project_Templates/templates/Meadow.StartKit/StartKit.Core/.template.config/template.json:13]: @jorgedevs 1. if we want our project templates to work with the --name parameter, then we need to add this as "sourcename", then rename our csproj files to 2. and add $safeprojectname$ to any .cs files (3.) where we want that replaced.

File: Meadow_DotNet_SDK/Project_Templates/templates/Meadow.StartKit/StartKit.Core/$safeprojectname$.Core.csproj [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Meadow.Sdk/1.1.0">
```
Comment (CartBlanche) [inline Meadow_DotNet_SDK/Project_Templates/templates/Meadow.StartKit/StartKit.Core/$safeprojectname$.Core.csproj:1]: @jorgedevs This is 2. note the file name

File: Meadow_DotNet_SDK/Project_Templates/templates/Meadow.StartKit/StartKit.Core/Commands/ChangeSetpointsCommand.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+﻿using Meadow.Cloud;
+
+namespace $safeprojectname$.Core
```
Comment (CartBlanche) [inline Meadow_DotNet_SDK/Project_Templates/templates/Meadow.StartKit/StartKit.Core/Commands/ChangeSetpointsCommand.cs:3]: @jorgedevs This is 3. Note how namespaces and uses and even classes or interfaces are named with the substitute variable.

**Review by jorgedevs — APPROVED** [review]
Reviewed but I wont merge until either @ctacke or @adrianstevens do a final look.

**Review by adrianstevens — APPROVED** [review]

**Pull request merged**